handlers/categoryHandler.js

```diff
@@ -0,0 +1,33 @@
+const Category = require('../model/categoryModel');
+
+// get all categories
+const allCategories = async (req,res) => {
+	const categoryLists = await Category.find();
+
+	if(!categoryLists){
+		return res.status(500).json({success:false})
+	}
```

Comment:
Please delete this lines of code. it will never run. and why did you write this ?